CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.1 (2019-03-20)
+
+- Fixed page summary component content to always fill its parent container. [#22](https://github.com/blackbaud/skyux-layout/pull/22)
+- Fixed action bar component to wrap to a second line when overflowing its action button container component. [#23](https://github.com/blackbaud/skyux-layout/pull/23)
```

Comment:
Would it be ok to just say "...to wrap when overflowing..."?

Reply:
I'll defer to @blackbaud-johnly on what is best there.

Reply:
Sorry. Just saw this. I'd been working on it and posted a suggestion in Slack. ... What do you guys think of changing this entry to "Fixed action button components to wrap when they overflow action button container component."?

Reply:
I like what is there now :)